src/main/java/seedu/address/logic/commands/ListInventoryCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_SUPPLY_ITEMS;
+
+import seedu.address.model.Model;
+
+/**
+ * Lists all supply items in the inventory.
+ */
+public class ListInventoryCommand extends Command {
+    public static final String COMMAND_WORD = "listInventory";
+
+    public static final String MESSAGE_SUCCESS = "List all supply items in inventory";
```

Comment:
Similarly, if the listAll command string can be changed, then perhaps this can be shortened to just "items" ?

Reply:
Yes that will be less confusing, btw can use the suggestions option to suggest and just commit here

Reply:
Ok noted

Reply:
```suggestion
    public static final String MESSAGE_SUCCESS = "List all items in inventory";
```